appveyor.yml

```diff
@@ -14,11 +14,14 @@ matrix:
 # what combinations to test
 environment:
   matrix:
-    - nodejs_version: 6
+    - nodejs_version: 8
+    - nodejs_version: 10
+
+platform:
+  - x64
 
 install:
-  - ps: Install-Product node $env:nodejs_version x64
-  - npm install -g npm@4
+  - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) $env:PLATFORM
```

Comment:
I invite you to make these improvements through a PR against the `v11-meta` branch. (Just like a regular PR, but selecting `v11-meta` instead of `master` as the target.)